pkg/controller/node/nodecontroller.go

```diff
@@ -371,6 +402,42 @@ func (nc *NodeController) Run() {
 			}
 		}, nc.nodeMonitorPeriod, wait.NeverStop)
 
+		go wait.Until(func() {
+			nc.evictorLock.Lock()
```

Comment:
yes, you're right - I completely forgot about that... In ideal world we'd just require UID to have a `String()` method and prefix the value with it, but we probably don't have time for it. namespace/pod-name sounds good then. Please add TODO for taking UID into account for deduplication.